App.js

```diff
@@ -0,0 +1,93 @@
+import React from "react"
+import Sidebar from "./components/Sidebar"
+import Editor from "./components/Editor"
+import { data } from "./data"
+import Split from "react-split"
+import {nanoid} from "nanoid"
+
+export default function App() {
+    const [notes, setNotes] = React.useState(JSON.parse(localStorage.getItem('Notes')) || [])
+    const [currentNoteId, setCurrentNoteId] = React.useState(
+        (notes[0] && notes[0].id) || ""
+    )
```

Comment:
If I understand well, this state will have the first note id as value if the first note exists or just an empty string. For a good readability, I can suggest you to format this code like this
```suggestion
    const [currentNoteId, setCurrentNoteId] = React.useState(
        notes[0] ? notes[0].id : ""
    )
```
In this case, if `note[0]` exists you return the Id else just an empty string